From 61026c85c363ae9499937a3a560aa3c7805660c5 Mon Sep 17 00:00:00 2001
From: Peng Fan <peng.fan@nxp.com>
Date: Tue, 13 Nov 2018 17:32:19 +0800
Subject: [PATCH 5106/5242] MLK-20347 ARM64: imx8qm: not assign HDMI for domu

commit  09586d25afeb09496b89a1030e92d2c638fda432 from
https://source.codeaurora.org/external/imx/linux-imx.git

HDMI power domain is under DC0, however DC0 is used by the first OS,
so domu has no permission to use HDMI.

Signed-off-by: Peng Fan <peng.fan@nxp.com>
Reviewed-by: Flynn xu <flynn.xu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-mek-dom0.dts     |    2 --
 .../boot/dts/freescale/fsl-imx8qm-mek-domu.dts     |    2 ++
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts
index 9822e0d..a180065 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts
@@ -96,8 +96,6 @@
 				SC_R_MIPI_1_I2C_0
 				SC_R_MIPI_1_I2C_1
 				SC_R_MIPI_1_PWM_0
-				SC_R_HDMI
-				SC_R_HDMI_I2C_0
 				SC_R_SDHC_0
 				SC_R_USB_0
 				SC_R_USB_0_PHY
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
index b9aab18..e90ded6 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
@@ -531,6 +531,7 @@
 };
 
 /delete-node/ &hdmi;
+/delete-node/ &hdmi_rx;
 /delete-node/ &irqsteer_dsi0;
 /delete-node/ &i2c0_mipi_dsi0;
 /delete-node/ &mipi_dsi_csr1;
@@ -611,6 +612,7 @@
 /delete-node/ &i2c0_cm40;
 /delete-node/ &i2c0_cm41;
 /delete-node/ &irqsteer_hdmi;
+/delete-node/ &irqsteer_hdmi_rx;
 /delete-node/ &i2c0_hdmi;
 
 &irqsteer_lvds1 {
-- 
1.7.9.5

